Documentation/ceph-cluster-crd.md

```diff
@@ -815,12 +815,25 @@ So just taking the `storage` section this will give something like:
           volumeMode: Block
           accessModes:
             - ReadWriteOnce
+      - metadata:
+          name: waldata
+        spec:
+          resources:
+            requests:
+              # Find the right size https://docs.ceph.com/docs/mimic/rados/configuration/bluestore-config-ref/#sizing
```

Comment:
Let's use links to the master documentation instead of mimic. Looks like we have a similar link above to fix as well.